draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -280,7 +280,7 @@
            </t>
             <t>
                 When an offerer or answerer indicates that it wants to establish a new DTLS association, it needs to make sure that
-                media packets in the existing DTLS association and new DTLS association can be de-multiplexed. In case

```

Comment:
I think we  agreed to say "media packets in any previously established DTLS association" not "any existing".